03/app.test.js

```diff
@@ -1 +1,25 @@
-import randomNumber from './app';
\ No newline at end of file
+import randomNumber from './app';
+
+it('Return 1 when min and max is 1', () => {
+  expect(randomNumber(1, 1)).toBe(1);
+});
+
+it('Throw an error when a non-number is provided', () => {
+  expect(() => {
+    randomNumber('X', 1).toThrow();
```

Comment:
Mają inny komunikat dla każdego przypadku będziesz mógł przetestować czy to jest ten którego się spodziewasz tj. `.toThrow(`message content`);`

Obecnie nie wiesz czy test jest poprawny, bo komunikat jest identyczny dla wszystkich.
Być może zawsze błąd dotyczy tego samego elementu, ale masz jak tego sprawdzić :)